lib/favicons.js

```diff
@@ -54,11 +55,13 @@ function generateIcons (loader, imageFileStream, pathPrefix, query, callback) {
     appName: query.appName
   }, function (err, result) {
     if (err) return callback(err);
-    var html = result.html.filter(function (entry) {
-      return entry.indexOf('manifest') === -1;
```

Comment:
But I need manifest.json in my app. How can we improve the quality of this file?